tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist =
+    black
+    flake8
+    isort
+    py{36,37}
+minversion = 1.9
```

Comment:
Why specify a `minversion`?

Reply:
Required for `skip_install` feature: https://tox.readthedocs.io/en/latest/config.html#conf-skip_install